dev_docs/tutorials/data/search.mdx

```diff
@@ -0,0 +1,490 @@
+---
+id: kibDevTutorialDataSearchAndSessions
```

Comment:
Can I view/preview this page somewhere?

Reply:
Not yet, this requires an additional PR in another repo